compiler/core/j.ml

```diff
@@ -75,7 +75,7 @@ and for_ident = ident
 and for_direction = Js_op.direction_flag
 and property_map = (property_name * expression) list
 and length_object = Js_op.length_object
-and delim = External_arg_spec.delim = DNone | DStarJ | DNoQuotes | DBackQuotes
+and string_kind = String_kind.t = Standard | Verbatim | RawJs | Template
```

Comment:
The current naming (DStarJ and such) is pretty bad, but I'm not a big fan of Standard and Verbatim, Verbatim is a double quoted string right? First time I hear about verbatim for strings but might be more common in other languages. But what about standard, what does it mean?